extension/dynamicconfig/model/model.go

```diff
@@ -0,0 +1,47 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Contains common models for the dynamic config service. The corresponding
+// Proto() methods convert the model representation to a usable struct for
+// protobuf marshalling.
+
+package model
```

Comment:
let's add a unit test of the model.go.

Reply:
Absolutely, in the process of writing them